src/pyOpenMS/tests/unittests/test_api_missing_methods.py

```
@@ -139,24 +143,62 @@ def test_get_spectrum_by_index(self):
+    def test_get_2d_peak_data_returns_arrays(self):
+        """get2DPeakData returns a 3-tuple of numpy float32 arrays."""
+        exp = pyopenms.MSExperiment()
+        spec = pyopenms.MSSpectrum()
+        spec.setMSLevel(1)
+        spec.setRT(1.0)
+        mzs = np.array([100.0, 200.0, 300.0], dtype=np.float64)
+        ints = np.array([1000.0, 2000.0, 3000.0], dtype=np.float32)
+        spec.set_peaks([mzs, ints])
+        exp.addSpectrum(spec)
+
+        # Query: RT range 0–10 s, m/z range 50–400, MS level 1
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)